src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.searchForm {
+  position: relative;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  min-width: 225px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchFormSm {
+  height: 42px;
+}
+
+.searchLabel {
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+}
+
+.searchInput {
+  width: 100%;
+  height: 95%;
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  border: none;
+  outline: none;
+  position: absolute;
+  left: 62px;
+}
+
+.searchInput::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.searchInputSm {
+  font-size: 14px;
+  line-height: 19px;
+  left: 33px;
+}
+
+.searchInputSm::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.icon {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  left: 26px;
+  top: 22px;
+}
+
+.iconSm {
+  position: absolute;
+  height: 19px;
```

Comment:
Don't repeat styles. Check 2nd topic in the checklist. And keep the same properties order